app/views/submissions/_submissions_display.html.erb

```diff
@@ -12,8 +12,10 @@
             <%= idx + 1 %>. <%= team.team_name %>
           </button>
         </h3>
-        <p class="text-muted">Students: <%= team.students.map {|student| student.user.user_name}.join(' & ') %></p>
-        <p class="text-muted">Advised by: <%= team.adviser ? team.adviser.user.user_name : 'Not assigned' %></p>
+        <div style="max-height: 42px">
```

Comment:
Should we make the div scrollable in the event when the text overflow the div?

Reply:
That's a good suggestion @JunWei96.